api/Element.json

```
@@ -6478,6 +6526,54 @@
           }
         }
       },
+      "previousElementSibling": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/previousElementSibling",
+          "support": {
+            "chrome": {
+              "version_added": "4"
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "firefox": {
+              "version_added": "3.5"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": "9"
+            },
+            "opera": {
+              "version_added": "10"
+            },
+            "opera_android": {
+              "version_added": "10.1"
+            },
+            "safari": {
+              "version_added": "4"
+            },
+            "safari_ios": {
+              "version_added": true
+            },
+            "samsunginternet_android": {
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": "≤37"
+            }
+          },
```

```suggestion
          "support": {
            "chrome": {
              "version_added": "1"
            },
            "chrome_android": {
              "version_added": "18"
            },
            "edge": {
              "version_added": "12"
            },
            "firefox": {
              "version_added": "3.5"
            },
            "firefox_android": {
              "version_added": "4"
            },
            "ie": {
              "version_added": "9"
            },
            "opera": {
              "version_added": "10"
            },
            "opera_android": {
              "version_added": "10.1"
            },
            "safari": {
              "version_added": "4"
            },
            "safari_ios": {
              "version_added": "3"
            },
            "samsunginternet_android": {
              "version_added": "1.0"
            },
            "webview_android": {
              "version_added": "≤37"
            }
          },
```

Same suggestions but in one go.